README.md

```diff
@@ -28,3 +28,35 @@ git submodule init
 git submodule sync --recursive
 git submodule update --recursive --init
 ```
+
+# Cloud Build CI
+
+This repository uses Cloud Build for continuous integration. The Cloud Build configuration file is located at [`cloudbuild.yaml`](cloudbuild.yaml).
+
+## Manually run the build
+
+Cloud Build can be triggered manually by running the following command
+from the root directory of this repository:
+
+```shell
+export GCP_PROJECT_TO_RUN_CLOUD_BUILD=<>
+export GKE_CLUSTER_NAME=<>
+export GKE_CLUSTER_LOCATION=<e.g. us-central1>
+export GIT_COMMIT_SHA=<>
+
+gcloud builds submit . \
```

Comment:
I've also found this command to be helpful for speeding up the iteration loop:

`cloud-build-local --config=cloudbuild.yaml --dryrun=false --substitutions=_CLUSTER_NAME=$GKE_CLUSTER_NAME,_ZONE=$GKE_CLUSTER_LOCATION .`

Reply:
In case it's helpful:

`sudo apt-get install google-cloud-sdk-cloud-build-local`